packages/react-router/rollup.config.js

```diff
@@ -69,6 +69,9 @@ module.exports = function rollup() {
         format: "esm",
         sourcemap: !PRETTY,
         banner: createBanner("React Router", version),
+        paths: {
+          "react-router": "./index.mjs",
+        },
```

Comment:
we rewrite the imports of `react-router` in the dom-export build to the most appropriate react-router entrypoint.